script.js

```diff
@@ -1,4 +1,289 @@
+// There will be only two players. One human and one computer (for the Base solution).
```

Comment:
Good job on writing pseudocode out. I hope this was helpful for you when you started actually coding and could use this as an easy reference while you coded